allennlp/nn/initializers.py

```diff
@@ -78,26 +118,33 @@ def __call__(self, module: torch.nn.Module) -> None:
                 not_explicitly_initialized_parameters.append((name, parameter))
 
         for name, parameter in not_explicitly_initialized_parameters:
-            self._default_initializer(parameter)
-            logger.info("Initializing %s using the Default "
-                        "Intitializer. (Normal(0, 1) unless user specified.)", name)
+            if any(re.search(exclude_regex, name) for exclude_regex in self._exclude):
+                logger.info("Excluding %s from default initialization", name)
+            else:
+                logger.info("Initializing %s using the Default "
+                            "Intitializer. (Normal(0, 1) unless user specified.)", name)
+                self._default_initializer(parameter)
 
     @classmethod
     def from_params(cls, params: Params) -> "InitializerApplicator":
         """
         Converts a Params object into an InitializerApplicator. The json should
         be formatted as follows:
 
-        initializers: {
-            parameter_regex_match1: {
-                "type": "normal"
-                "mean": 0.01
-                "std": 0.1
-            },
-            parameter_regex_match2: "uniform",
+        {
+            "initializers": {
+                "[parameter_regex_match1]": {
```

Comment:
Why are there brackets here? It doesn't look like the functionality has changed.